templates/app-scaffold/ts/.github/workflows/push_production.workflow.yml

```diff
@@ -0,0 +1,95 @@
+name: Push checks
+
+on:
+  push:
+    branches:
+      - production
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - uses: actions/setup-node@v1.4.4
+        with:
+          node-version: "12"
+      - uses: ./.github/actions/backend-lint
+
+  functional-tests:
+     name: Functional Tests
+     runs-on: ubuntu-18.04
+     needs: [lint]
+     steps:
+        - uses: actions/checkout@v2
+        - name: Cache node modules
+          uses: actions/cache@v2
+          env:
+            cache-name: cache-node-modules
+          with:
+            path: ~/.npm
+            key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+            restore-keys: |
+              ${{ runner.os }}-build-${{ env.cache-name }}-
+              ${{ runner.os }}-build-
+              ${{ runner.os }}-
+        - uses: actions/setup-node@v1.4.4
+          with:
+            node-version: "12"
+        - uses: ./.github/actions/backend-functional-test
+
+  deploy-kuzzle:
+     name: Deploy Kuzzle
+     runs-on: ubuntu-18.04
+     needs: [functional-tests]
+     steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+        - uses: docker/setup-buildx-action@v1
+        - name: Login to DockerHub
+          uses: docker/login-action@v1
+          with:
+            registry: registry.gitlab.com
+            username: ${{ secrets.GITLAB_DOCKER_USERNAME }}
+            password: ${{ secrets.GITLAB_DOCKER_TOKEN }}
+        - name: Build and push image
+          uses: docker/build-push-action@v2
+          with:
+            context: ./backend
+            file: ./backend/Dockerfile
+            push: true
+            tags: registry.gitlab.com/kuzzle/<CHANGE-ME>:latest
```

Comment:
Is there a place where we give instructions to come here and update these **<CHANGE-ME>** ?

Reply:
No, but as we agreed, the deployment part is now removed